src/user/SignUpForm.js

```diff
@@ -0,0 +1,80 @@
+// Sign Up Form allows anyone to sign up for an account to access their personal dashboards
+// The contact form requests First Name, Last Name, Email, Password & Confirm Password.
+// The sign up form then redirects the user to their personal dashboards
+
+import React from 'react';
+import {Form, Button} from 'react-bootstrap';
+
+class SignUpForm extends React.Component {
```

Comment:
See above

Reply:
Agreed, I like the convention. I saw some places use React.Component so kind of got used to it, but then the "React" part is kind of redundant and not DRY. I will use this from now on!